cumulusci/tasks/salesforce/tests/test_activate_flow.py

```diff
@@ -0,0 +1,88 @@
+import unittest as test_activate_flow
```

Comment:
What's the purpose for aliasing `unittest`? This is generally not something you need to do.

Reply:
Agreed I thought I had read David Glick had asked for it didn't really make sense to me but didn't really impact my life so i did it.

Reply:
This is definitely not what I was asking for. I was asking you to rename the file; I put my comment on this line because I needed to put it somewhere.